src/gtimelog/tests/test_timelog.py

```diff
@@ -42,7 +42,7 @@ def check_output(self, want, got, optionflags):
 def doctest_as_hours():
     """Tests for as_hours
 
-        >>> from gtimelog.timelog import as_hours
+        >>> from gtimelog.utils import as_hours
         >>> from datetime import timedelta
```

Comment:
Test modules should mirror code modules.  For this reason it may be better to do the switch to pytest before we start moving modules around.